packages/react-img-viewer/src/types.ts

```diff
@@ -0,0 +1,23 @@
+// src/types.ts
+export enum ImageViewerMode {
+    inline = "inline",
+    fullScreen = "fullScreen",
+  }
```

Comment:
Instead of mode we should define different types for ImageViewer and FullScreenViewer.